next-sitemap.js

```diff
@@ -0,0 +1,24 @@
+const SITE_NAME = 'https://notpoisonousmushrooms.com'
+module.exports = {
+  siteUrl: process.env.SITE_URL || SITE_NAME,
```

Comment:
When would you use `process.env.SITE_URL`?
(Especially because lines 6 and 10 only have `SITE_NAME`)

Reply:
I left this in on purpose to remind myself that this file has access to env vars.
I can remove it if you'd prerfer - it just works as a memo here.

Reply:
Add process.env on line 1